packages/viewer/src/components/Shared/Source.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+  export let source: string
+  let showMarkdown = false
+</script>
+
+<button
+  class="btn btn-secondary btn-sm mb-3"
```

Comment:
I think the style of the button is a little visually intrusive at the moment. Could you consider replacing `btn-secondary` with `{showMarkdown ? 'btn-secondary' : 'btn-outline-secondary'}` as in [FilterButtons.svelte](https://github.com/pi-base/web/blob/main/packages/viewer/src/components/Traits/FilterButtons.svelte)?

Reply:
good call